input/fsh/canonical/COVIDRegularHealthCheckQuestionnaire.fsh

```diff
@@ -0,0 +1,348 @@
+Alias: $usage-context-type = http://terminology.hl7.org/CodeSystem/usage-context-type
+Alias: $sct = http://snomed.info/sct
+Instance: COVIDRegularHealthCheckQuestionnaire
+InstanceOf: Questionnaire
+Usage: #definition
+
+* identifier[0].use = #official
+* identifier[=].value = "COVIDRegularHealthCheckQuestionnaire"
+* identifier[=].period.start = "2023-07-19"
+* identifier[+].use = #temp
+* identifier[=].value = "QuestionTemplate-CitC-COVID19-RegularHealthCheck"
+* identifier[=].period.end = "2023-07-19"
+
+* version = "0.1.5"
+* name = "COVIDRegularHealthCheckQuestionnaire"
+* title = "COVID19 Initial Health Assessment Questionnaire"
+* status = #draft
+* subjectType = #Patient
+* publisher = "Te Whatu Ora"
+* description = "Questions about a patient's recent COVID-19 history"
```

Comment:
Not currently matching

Reply:
hmm on my view of the branch, it shows correctly as 
* description = "Gather the status of a patient who is self isolating"

How can the PR look different to the branch I created it on?

Reply:
The seed resource description should match the IG one "Gather the status of a patient who is self isolating"